src/utils/__tests__/githubApi.test.ts

```diff
@@ -161,6 +162,7 @@ describe('retryHttp', () => {
         ...params,
         cleanupFn: async () => {
           cleanupCalled += 1;
+          return Promise.resolve();
```

Comment:
Isn't this done implicitly?

Reply:
Yes - but the linter was mad, so instead of ignoring it, I f figured, why not be explicit? ¯\_(ツ)_/¯ Happy to switch to a linter-calming comment if you'd like.

Reply:
It seems that we use different linters, then 😅

Reply:
🤔

![image](https://user-images.githubusercontent.com/14812505/80394799-3c209a00-8867-11ea-9bef-484107c64811.png)

vs

![image](https://user-images.githubusercontent.com/14812505/80394911-5a869580-8867-11ea-8c90-a4f7441e4aac.png)

Reply:
Don't know why our `yarn lint` command doesn't catch this now.
And I think `return undefined` should also work.

Reply:
> Don't know why our yarn lint command doesn't catch this now.

Yeah, that is odd.

> And I think return undefined should also work.

It doesn't like `return undefined`, but it's okay with `return`. But at that point, if we're bothering to return anything, why not return the thing the signature says we should return?

Reply:
Return values of async functions are wrapped in Promises automatically, so normally you don't have to return `Promise.resolve(...)`, value is enough.
In any case, please let's try to unify the linting configuration, it seems that your VS code does more than mine and than `yarn lint`.

Reply:
> Return values of async functions are wrapped in Promises automatically

True, and fair point. Maybe it's the remnants of the Python dev still in me, LOL?

![image](https://user-images.githubusercontent.com/14812505/80480063-dda9f900-8904-11ea-8a2c-60d193735cee.png)

As for linting, if we're going to work on it at all we should probably use it as an excuse to migrate to`eslint`. (See https://medium.com/palantir/tslint-in-2019-1a144c2317a9.) I'll make an issue.